### PR TITLE
Fix configure typo and update doc of loading history data

### PR DESCRIPTION
**Pull request by amoskong**
The 1st patch fixed a typo of ip address in prometheus.yml

The 2nd patch updated README about loading history data to prometheus server. User may report a issue with a prometheus data, we also download prometheus data in SCT performance testing, it's a easy way to review the history data.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,19 @@ Update `prometheus/prometheus.yml` with the targets (server you wish to monitor)
 ./start-all.sh
 ```
 
+### Load original data to prometheus server
```
Comment (tzach) [inline README.md:32]: @kongove can you explain where this data comes from?
Is it the same data capture in this procedure:
https://github.com/scylladb/scylla/wiki/How-to-report-a-Scylla-problem#prometheus

File: prometheus/prometheus.yml [hunk 2]
```diff
@@ -10,4 +10,4 @@ scrape_configs:
 - job_name: scylla
   honor_labels: true
   static_configs:
-  - targets: ['127.0.0.1.18:9103']
+  - targets: ['127.0.0.1:9090']
```
Comment (tzach) [inline prometheus/prometheus.yml]: why 9090? port 9103 is the default  collectd_exporter port, as describe here
https://github.com/scylladb/scylla/wiki/Monitor-Scylla-with-Prometheus-and-Grafana#collectd_exporter

**Comment by tzach** [comment]
@kongove thanks for the PR, it make a lot of sense
Two comments above

File: README.md (hunk 1 above)
Comment (amoskong) [reply README.md:32]: Yes, it's same. I loaded the prometheus data which got from SCT performance test, grafana can present it correctly. It's helpful.

Data source:
1. Download from docker prometheus server, reference:
https://github.com/scylladb/scylla/wiki/How-to-report-a-Scylla-problem#prometheus
2. Get from Scylla-Cluster-Test log.
3. etc

File: prometheus/prometheus.yml (hunk 2 above)
Comment (amoskong) [reply prometheus/prometheus.yml]: I referenced the example in prometheus github [1]. I will keep using 9103 port when I rebase the PR. 

[1] https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus.yml

File: README.md (hunk 1 above)
Comment (tzach) [reply README.md:32]: Thanks, please added this info to the README itself, so users will make the connection, and can go back to Scylla wiki. Its not obvious to everyone

**Pull request merged**